src/components/PanelsWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC, useEffect } from 'react';
+import Panel from '@/components/panel/Panel';
+import { useConfig } from '@/contexts/ConfigContext';
+import { getManifestUrl, getCollectionUrl } from '@/utils/panel';
+
+const PanelsWrapper: FC = ({}) => {
+  const { config, openedPanels, setOpenedPanels } = useConfig();
+
+  function initOpenedPanels(panels: Panel[]) {
```

Comment:
Is this wrapper function needed? You could just call `setOpenedPanels` directly.

Reply:
yes, true. I removed initOpenedPanels()